### PR TITLE
Remove resumed state

### PR DESCRIPTION
**Pull request by mbj**
Extraction from #398

### CODE REVIEW

**Comment by mbj** [comment]
@dkubb This is ready for review.

**Comment by dkubb** [comment]
@mbj when this is applied to mrh-spree can we include a migration to remap existing orders to 'canceled'` and then update the enum type altogether so it's not even possible?

File: core/app/models/spree/order.rb [hunk 1]
```diff
@@ -283,9 +274,7 @@ def outstanding_balance?
       self.outstanding_balance != 0
     end
 
-    def can_ship?
-      complete? || resumed?
-    end
+    alias_method :can_ship?, :complete?
```
Comment (dkubb) [inline core/app/models/spree/order.rb]: Can we have a follow-up PR that replaces all instances of `can_ship?` to `complete?` and removes this altogether?

**Comment by mbj** [comment]
> @mbj when this is applied to mrh-spree can we include a migration to remap existing orders to 'canceled'` and then update the enum type altogether so it's not even possible?

Yeah, thats the plan.

File: core/app/models/spree/order.rb (hunk 1 above)
Comment (mbj) [inline core/app/models/spree/order.rb]: Yes.

**Pull request merged**